api4/emoji.go

```diff
@@ -83,15 +82,14 @@ func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	emoji := model.EmojiFromJson(strings.NewReader(props["emoji"][0]))
-	if emoji == nil {
+	var emoji model.Emoji
+	if jsonErr := json.Unmarshal([]byte(props["emoji"][0]), &emoji); jsonErr != nil {
 		c.SetInvalidParam("emoji")
-		return
```

Comment:
Why doesn't the code return here any longer?

Reply:
Thanks, that looks like a mistake.